.gitlab-ci.yml

```diff
@@ -67,6 +68,23 @@ variables:                         &default-vars
     paths:
       - ./artifacts/
 
+#### stage:                        lint
+
+fmt:
+  stage:                           lint
+  <<:                              *docker-env
```

Comment:
`&docker-env` refers to `linux-docker` runners - we use them for heavyweight jobs (e.g. compilation stuff). These runners have limited capacity. Please resort to `&kubernetes-env` (`kubernetes-parity-build`) instead.